nano/node/bootstrap/bootstrap.hpp

```diff
@@ -203,9 +231,12 @@ class bootstrap_limits final
 	static constexpr double bootstrap_minimum_elapsed_seconds_blockrate = 0.02;
 	static constexpr double bootstrap_minimum_frontier_blocks_per_sec = 1000.0;
 	static constexpr unsigned bootstrap_frontier_retry_limit = 16;
+	static constexpr unsigned bootstrap_lazy_retry_limit = bootstrap_frontier_retry_limit * 10;
 	static constexpr double bootstrap_minimum_termination_time_sec = 30.0;
 	static constexpr unsigned bootstrap_max_new_connections = 10;
 	static constexpr unsigned bulk_push_cost_limit = 200;
 	static constexpr std::chrono::seconds lazy_flush_delay_sec = std::chrono::seconds (5);
+	static constexpr unsigned bootstrap_lazy_destinations_request_limit = 200;
+	static constexpr std::chrono::seconds lazy_destinations_flush_delay_min = std::chrono::minutes (2);
```

Comment:
Does `_min` here mean minutes or minimum? because the type is `seconds`, bit confusing as above a variable has the `_sec` suffix and the type is seconds.

Reply:
Updates to lazy_destinations_flush_delay_minutes